commands/cardcreator.js

```diff
@@ -113,13 +113,14 @@ class CardCreator {
     }
 
     const cardFolder = `${this._customCardsDir}/${cardFolderName}`;
-    if (fs.existsSync(templateCardFolder)) {
+    const themeCardFolder = path.join(this.config.dirs.themes, defaultTheme, templateCardFolder);
+    if (fs.existsSync(themeCardFolder)) {
```

Comment:
Do we want to check if the file exists at the templateCardFolder first? That way checking inside the theme is the fallback rather than the only location which is checked. If we don't check the templateCardFolder first, then I think we lose the ability to create a card based on a top-level card

Reply:
wow I didn't know we could do that! top level cards are not included in jambo describe (in this PR or before), but I can fix that here

Reply:
Oh, maybe it's not officially supported (i'm also not sure if anyone uses that functionality). However I do remember Bryan saying that in general for describe and commands, that our logic should first check at the top level and then check the theme as a fallback

Reply:
sure makes sense, i can imagine somebody wanting to copy a top level card and make small tweaks

Reply:
actually hMMmmM it's possible they actually don't want the extra options (thinking of something like yanswers with its 20 different cards), just asked Tom what his opinion his

Reply:
Tom gave the ok!